app/src/main/java/com/bogdan/codeforceswatcher/adapter/ContestAdapter.kt

```diff
@@ -48,8 +48,8 @@ class ContestAdapter(private var items: List<Contest>, private val ctx: Context)
         notifyDataSetChanged()
     }
 
-    private fun getDateTime(seconds: Long): String {
-        return SimpleDateFormat("kk:mm MMM d, yyyy", Locale.ENGLISH).format(Date(seconds * 1000)).toString()
+    private fun getDateFromSeconds(seconds: Long): String {
```

Comment:
Here you have duplication `FromSeconds` and `seconds`. `getDateTime` was better in this case. If you want to specify that you pass seconds you can use named arguments, like `getDateTime(seconds = lastRatingChange.ratingUpdateTimeSeconds)`.